README.md

```diff
@@ -1,2 +1,59 @@
 # fiware-iot-stack
 Repository to store and manage end user documentation related to the IoT Stack FIWARE based 
+
+## How to generate documentation for a specific release
+
+Assuming you are in master branch with everything ready to release (v4.2 is a fictional example):
+
+* Create the release branch:
+
+```
+git checkout -b release/v4.2
+```
+
+* Adjust URLs to point to the right version of the components included in that release. This is
+  done automatically by the `prepare_release.py` using as input the repo-to-version configuration
+  file
+
+```
+./prepare_release.py -v -d docs/ -c iotpv4.2.conf
```

Comment:
Since this a Python script, I think it should be mentioned that Python is required :)

Event, the packages needed. For instance, I'm getting the following error:

```
$ ./prepare_release.py -v -d docs/ -c iotpv4.2.conf
Traceback (most recent call last):
  File "./prepare_release.py", line 31, in <module>
    import configparser
ImportError: No module named configparser
```